internal/aws/cwlogs/cwlog_client.go

```diff
@@ -57,7 +57,11 @@ func NewClient(logger *zap.Logger, awsConfig *aws.Config, buildInfo component.Bu
 func (client *Client) PutLogEvents(input *cloudwatchlogs.PutLogEventsInput, retryCnt int) (*string, error) {
 	var response *cloudwatchlogs.PutLogEventsOutput
 	var err error
-	var token = input.SequenceToken
+	// CloudWatch Logs API was changed to ignore the sequenceToken
+	// PutLogEvents actions are now accepted and never return
+	// InvalidSequenceTokenException or DataAlreadyAcceptedException even
+	// if the sequence token is not valid.
+	token := aws.String("Noop")
```

Comment:
It is not necessary anymore, but if we were to eliminate it completely instead of sending a dummy token, I would need to refactor `cwlog_client` and `pusher` modules.

Reply:
Ack, can we add a comment stating this?

Reply:
I went ahead and refactored what was needed.